themes/_core/js/facets.js

```diff
@@ -23,52 +23,53 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
 import $ from 'jquery';
+import prestashop from 'prestashop';
 
 let currentRequest = null;
 
 function updateResults(data) {
-    prestashop.emit('updateProductList', data);
-    window.history.pushState(data, document.title, data.current_url);
+  prestashop.emit('updateProductList', data);
+  window.history.pushState(data, document.title, data.current_url);
 }
 
 function handleError(xhr, textStatus) {
-    if (textStatus === 'abort') {
-        return;
-    }
-    // TODO: feedback
+  if (textStatus === 'abort') {
+    console.warn('Request aborted');
```

Comment:
Is it wanted the behavior change ?

Reply:
Changed it into the correct behavior, idk why my conflict was empty, so I decided to put another behavior, but thanks to you I seen the past one